recipe/meta.yaml

```diff
@@ -19,34 +20,36 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - hatchling
     - hatch-requirements-txt
     - hatch-fancy-pypi-readme >=22.5.0
     - pip
   run:
-    - python >=3.8
+    - python >={{ python_min }}
     - aiofiles >=22.0,<24.0
     - anyio >=3.0,<5.0
-    - fastapi
+    - audioop-lts <1.0
```

Comment:
```suggestion
```
https://github.com/conda-forge/staged-recipes/pull/28300
Won't be avalaible on Conda-Forge until the end of Python 3.13 migration. I made a PR, but can't work since Staged-Recipe currently can't build 3.13 packages.

Reply:
Which also mean this can't be added to a python noarch recipe except if you intent to only have it avalaible for 3.13